demonstrations/tutorial_qnn_multivariate_regression.py

```diff
@@ -0,0 +1,253 @@
+# process to train the variational circuit will be performed using JAX. Checkout [#demojax]_
+# to learn more about
+# how to use JAX to optimize your QML models.
+# .. figure:: ../_static/demonstration_assets/qnn_multivariate_regression/qnn_diagram.jpg
+#   :align: center
+#   :width: 90%
+
+@jax.jit
+def mse(params,x,targets):
+    return (quantum_neural_network(params,x)-jnp.array(targets))**2
+@jax.jit
+def loss_fn(params, x,targets):
+    mse_pred = jax.vmap(mse,in_axes=(None, 0,0))(params,x,targets)
+    loss = jnp.mean(mse_pred)
+    return loss
+
+####################################################################### 
+#Here, we are choosing a learning rate of 0.05 and 200 steps.
+
+opt = optax.adam(learning_rate=0.05)
+max_steps=200
+
+@jax.jit
+def update_step_jit(i, args):
+    # Step we loop over to optimize the trainable parameters
+    params, opt_state, data, targets, print_training = args
+    loss_val, grads = jax.value_and_grad(loss_fn)(params, data, targets)
+    updates, opt_state = opt.update(grads, opt_state)
+    params = optax.apply_updates(params, updates)
+
+    def print_fn():
+        jax.debug.print("Step: {i}  Loss: {loss_val}", i=i, loss_val=loss_val)
+    # if print_training=True, print the loss every 50 steps
+    jax.lax.cond((jnp.mod(i, 50) == 0 ) & print_training, print_fn, lambda: None)
+    return (params, opt_state, data, targets, print_training)
+
+@jax.jit
+def optimization_jit(params, data, targets, print_training=False):
+    opt_state = opt.init(params)
+    args = (params, opt_state, jnp.asarray(data), targets, print_training)
+    # We loop over max_steps iterations to optimize the parameters
+    (params, opt_state, _, _, _) = jax.lax.fori_loop(0, max_steps+1, update_step_jit, args)
+    return params
+
+######################################################################
+# Now we will train the variational circuit with four layers and obtain a vector :math:`\vec{\theta}` with the optimized parameters. At every 10 steps, the code prints out the value of the loss function.
+
+wires=2
+layers=4
+params_shape = qml.StronglyEntanglingLayers.shape(n_layers=layers,n_wires=wires)
+params=np.random.default_rng().random(size=params_shape)
+best_params=optimization_jit(params, x_train, jnp.array(y_train), print_training=True)
+
+######################################################################
+# Once the optimized :math:`\vec{\theta}` has been obtained, we can use those parameters to build our fitted version of the function.
+
+def evaluate(params, data):
+    y_pred = jax.vmap(quantum_neural_network, in_axes=(None, 0))(params, data)
+    return y_pred
+y_predictions=evaluate(best_params,x_train)
+
+######################################################################
+# To compare the fitted function to the exact target function, let's take a look at the :math:`R^2` score.
+
+from sklearn.metrics import r2_score
+r2 = round(float(r2_score(y_train, y_predictions)),3)
+print("R^2 Score:", r2)
```

Comment:
To print it with latex:
```suggestion
print(r"R^2 Score:", r2)
```